tgui/packages/tgui/interfaces/CrewMonitor.js

```diff
@@ -118,10 +118,10 @@ const CrewMonitorDataView = (_properties, context) => {
           <Table.Cell>Status</Table.Cell>
           <Table.Cell>Location</Table.Cell>
         </Table.Row>
-        {crew.filter(searcher).map((cm) => {
+        {crew.filter(searcher).map((cm, index) => {
           const highlighted = highlightedNames.includes(cm.name);
           return (
-            <Table.Row key={cm.name} bold={!!cm.is_command}>
+            <Table.Row key={index} bold={!!cm.is_command}>
```

Comment:
I think it will be fine, similar to PRs like #21528. Its not a great solution, but it does get the job done usually. I think we should try this, and if it doesn't work we can look into other solutions.